chapters/en/Unit 1 - Fundamentals/Chapter 1 - Fundamentals of Computer Vision/defination.mdx

```diff
@@ -1,36 +1,16 @@
 # What Is Computer Vision
 
-Let's revisit our example from the previous chapter: kicking a ball. As we have seen,
-this involves multiple tasks that our brains can do in a split second. The extraction of
-what is meaningful information from an image input is at the core of computer vision.
-But what is computer vision?
+Let's revisit our example from the previous chapter: kicking a ball. As we have seen, this involves multiple tasks that our brains can do in a split second. The extraction of what is meaningful information from an image input is at the core of computer vision. But what is computer vision?
 
 ## Definition
-Computer vision is a multi-disciplinary subfield of artificial intelligence (AI)  
-that enables computer systems to analyze, interpret, and process visual information.
-It involves the development of algorithms, models, and systems that can generate images or extract knowledge from images,
-videos, and other forms of visual data.
 
-## Deep learning and the computer vision renaissance
-
-The evolution of computer vision has been marked by a series of incremental advancements in
-and across its interdisciplinary fields, where each step forward gave rise to breakthrough algorithms,
-hardware, and data, giving it more power and flexibility. One such history was the jump to the widespread
-use of deep learning methods.
+Computer vision is the science and technology of making machines see. It involves the development of theoretical and algorithmic methods to acquire, process, analyze, and understand visual data, and to use this information to produce meaningful representations, descriptions, and interpretations of the world. (*Forsyth & Ponce, Computer Vision: A Modern Approach*)
```

Comment:
I've taken a textbook definition and added it here based on johko comment

Reply:
I quite like the definition we had before, too. I think we can improve it to mix both in. It shouldn't be too hard to find a reference for the first one too.

Reply:
I'm quite sure that we cant quote the reference for the previous definition because it's funny but i copied like 5-6 definitions on a notepad and added pieces together so.......
I agree our previous definition is good but this what they are expecting right?